### DISCUSSION TITLE
nouvelle idée de mini event

### DISCUSSION THREAD

**Discussion by romain22222** [Crownicles/Crownicles#930] [Nouvelles idées]
quand tu tombes dessus : un truc chiant mais qui n'impacte pas la progression du joueur

exemple : Pendant votre marche, vous voyez un magnifique oiseau passer au dessus de vous. Vous le trouviez magnifique jusqu'à ce qu'il vous lâche une fiante sur la banane. Dégouté, vous continuez votre route.

**Comment by BastLast** [comment]
juste des nouvelles phrases pr le nothing happenned en gros ?

**Reply by romain22222** [reply]
par la j'entendais un mini event dont l'emote s'adapte à la phrase proposée, genre la ce serait un visage paconten

**Reply by BastLast** [reply]
hm j'ai du mal à voir l'interet d'un mini event à part juste pr un émoji

**Reply by romain22222** [reply]
et bien du coup ce serait une évolution du doNothing, car il est relativement vide et ignoré pour le moment

**Reply by BastLast** [reply]
faudra voir on en rediscutera
